docs/api.md

```diff
@@ -403,6 +403,26 @@ Logging out is not strictly necessary for Web Users; all sessions expire 24 hour
 + Response 403 (application/json)
     + Attributes (Error 403)
 
+#### Logging out current session [DELETE /v1/sessions/current]
+
+This endpoint causes the current session to log itself out. Logging out is not strictly necessary for Web Users; all sessions expire 24 hours after they are created. But it can be a good idea, in case someone else manages to steal your token.
+
+Only the session that was used to authenticate the request is logged out. If the Actor associated with the session has other sessions as well, those are not logged out.
+
++ Request
+    + Headers
+
+            Authorization: Bearer lSpAIeksRu1CNZs7!qjAot2T17dPzkrw9B4iTtpj7OoIJBmXvnHM8z8Ka4QPEjR7
```

Comment:
For web users, the token will also be sent as a cookie, and perhaps in more cases.  Would it be helpful to also allow supplying the token-to-delete via cookie?

Reply:
The token could be supplied via a cookie. Frontend might do that. However, we don't encourage API users to use cookie auth.